security/advancedtls/internal/testutils/testutils.go

```diff
@@ -79,12 +89,18 @@ func (cs *CertStore) LoadCerts() error {
 	if cs.ClientCert2, err = tls.LoadX509KeyPair(testdata.Path("client_cert_2.pem"), testdata.Path("client_key_2.pem")); err != nil {
 		return err
 	}
+	if cs.ClientCert3, err = tls.LoadX509KeyPair(testdata.Path("crl/provider/client_cert.pem"), testdata.Path("crl/provider/client_cert.key")); err != nil {
+		return err
+	}
```

Comment:
Similar, but... loops.

Also why does this break the existing pattern?

Is it possible to keep using the old one?  Also note that a new directory for new files means more work when we import into g3, so it would be nice to keep everything together if possible.

Reply:
There is the same problem with existing `testdata` and `testdata/crl` dirs - Readme files don't contain the exact commands and don't cover all the files. That's why I created a clean subdir with better documentation inside.
If import into g3 is tough I can utilize an existing dir but add suffix/prefix for new files - WDYT?

Reply:
It would be even nicer to add documentation and/or scripts where they don't exist, rather than create a new directory where things are better.  Unless you're taking an AI to do that as a cleanup.

Different filenames is fine.  I just know creating new directories for data files incurs an extra cost on our import process, and using data files is different in OSS vs. internally.  So I'd rather avoid it.

Reply:
Moved everything under existing `testdata/crl` directory, each file has a `provider_` prefix, merged Readme files